### DISCUSSION TITLE
Saved search filters

### DISCUSSION THREAD

**Discussion by joshuadanpeterson** [pieces-app/support#186] [Ideas]
I think being able to save search filters would be a useful feature. That way, we could easily access certain snippets with a click of a button. It'd make the search process a whole lot easier than having to retype filters every time we wanted to use them.

**Comment by pythonistical** [comment]
What will the UI look like?

**Reply by joshuadanpeterson** [reply]
<img width="1102" alt="Screenshot 2024-05-04 at 15 55 49" src="https://github.com/pieces-app/support/assets/6033094/7f7bcaa4-8bce-4998-9136-3c218236fd48">
It could just be something as simple as a "Saved Filters" button on the Add Search Filters page that pulls up a list of saved filters

**Comment by pythonistical** [comment]
I think it's a fine idea to work on.
